community/sinit/post-install

```diff
@@ -1,33 +1,35 @@
 #!/bin/sh -e
 
-cat <<EOF
-
-IMPORTANT:
-        This package is installed as /usr/bin/sinit
-        Please update your kernel parameters
-        accordingly,
+grep -q SINIT_ /etc/rc.conf || cat <<EOF >> /etc/rc.conf
```

Comment:
I didn't know you have talked about this, but I really don't think that this is an issue. We could just inform the user that we have added these comments to their rc.conf. Since these are just comments, the only impact this has to the user's system is that it saves the user the time to copy and paste it on their own.

Reply:
TBH I can't remember exactly how it all happened. The change is fine with me, personally. I'm just not sure it's a best practice however to just append things to peoples config files in /etc/

Reply:
Yeah, it is not the best practice, but since it is only a comment, it can be justified. 

We could also modify init and make it read every `*.conf` file on `/etc/rc.d`, the way we do it for hooks. That way we could simply drop an `sinit.conf` to `/etc/rc.d` without directly editing the user's personal configurations.

Reply:
I'm against modifying user config files like this. I'd much rather prefer a post-install message. :+1: